assets/app/view/game/round/auction.rb

```diff
@@ -104,16 +104,38 @@ def render_companies
           @selected_company = @step.auctioning if @step.auctioning
 
           companies = @step.available.select(&:company?)
-          if @step.respond_to?(:tiered_auction?) && @step.tiered_auction?
-            companies.group_by(&:value).values.map do |tier|
-              h(:div, { style: { display: 'table' } }, tier.map { |company| render_company(company) })
+          rendered_companies = if @step.respond_to?(:tiered_auction?) && @step.tiered_auction?
+                                 companies.group_by(&:value).values.map do |tier|
+                                   h(:div, { style: { display: 'table' } }, tier.map do |company|
+                                                                              render_company(company)
+                                                                            end)
+                                 end
+                               else
+                                 companies.map { |company| render_company(company) }
+                               end
+
+          if @step.respond_to?(:unavailable_companies) && @step.unavailable_companies
+            rendered_unavailable_companies = @step.unavailable_companies.map do |company|
+              render_company(company, false)
             end
+            unvailable = h(:div, [
+              h(:div, {
+                  style: {
+                    "font-weight": 'bold',
+                    "font-size": '1.1rem',
+                    "padding-top": '3rem',
```

Comment:
There’s `.bold` in main.css, but maybe you just want h3 instead of div => bold + font-size of 1.1rem and it’s semantic, too.
Also please use camelCase for css-_attributes_: `paddingTop`.

Reply:
its removed either way but thanks ;)